chromium_src/third_party/blink/renderer/core/frame/local_dom_window.cc

```diff
@@ -4,11 +4,38 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "third_party/blink/renderer/core/frame/local_dom_window.h"
+#include "brave/third_party/blink/renderer/core/brave_session_cache.h"
+#include "third_party/blink/renderer/core/events/mouse_event.h"
+#include "third_party/blink/renderer/core/events/pointer_event.h"
+
+#define outerHeight outerHeight_ChromiumImpl
+#define outerWidth outerWidth_ChromiumImpl
+#define screenX screenX_ChromiumImpl
+#define screenY screenY_ChromiumImpl
 
 #include "src/third_party/blink/renderer/core/frame/local_dom_window.cc"
 
+#undef outerHeight
+#undef outerWidth
+#undef screenX
+#undef screenY
+
 namespace blink {
 
+using brave::FarbleKey;
+
+int LocalDOMWindow::MaybeFarbleInteger(brave::FarbleKey key,
```

Comment:
I think we need some RenderViewTests for this stuff

Reply:
I have revised some of this code by factoring out helper functions. Currently farbled APIs are covered by browser tests.

Reply:
We discussed this and decided to postpone for now.